common/common.go

```diff
@@ -14,28 +16,45 @@ var log = logrus.WithField("package", "common")
 
 func DownloadFile(url, destinationDir string) (string, error) {
 	log.WithField("url", url).Debug("downloading file")
+	var name string
+	var err error
+	var retry bool
+	for a := 1; a <= 3; a++ {
+		name, err, retry = downloadFile(url, destinationDir)
+		if err != nil && retry {
+			log.WithError(err).Debug(fmt.Sprintf("Attempt %d failed", a))
```

Comment:
I think there's a `Debugf` function that you can use instead of calling `Sprintf`.

Reply:
good call, switched to that